client/src/components/common/Card/index.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import moment from "moment";
+import { Link } from "react-router-dom";
+
+import "./style.css";
+
+const Card = ({ info }) => {
+  const { image, title, category, date, event_code } = info;
+  return (
+    <li className="card">
+      <img className="card__image" src={image} alt={title} />
+      <div className="card__content">
+        <h3 className="card__p">
+          <b>{title}</b>
+        </h3>
+        <p className="card__p">
+          <b>By :</b> {category}
```

Comment:
Personally I'd remove the labels `By :` `Date :` etc, but that's just preference